controllers/activities.js

```diff
@@ -78,10 +78,13 @@ module.exports = function(app) {
     });
 
     // DELETE activity - yes, can remove from itinerary
-    app.delete('/itineraries/:itinId/activities/:actId', function (req, res) {
+    app.get('/itineraries/:itinId/activities/:actId/delete', function (req, res) {
```

Comment:
Is it better to avoid convention for something that works? If we do it this way, people will be able to delete itineraries using links.